lib/app_generator/search_app.rb

```diff
@@ -13,6 +13,7 @@ class SearchApp < App
                   :garbagecollectioninterval => :garbagecollectioninterval,
                   :doc_type => :doc_type,
                   :threads_per_search => :threads_per_search,
+                  :allowedlidbloat => :allowedlidbloat,
```

Comment:
Consider using snake case (as elsewhere): allowed_lid_bloat

Reply:
Snake it is.